From f407aa1462bd802dbe210da998b81936c870b78e Mon Sep 17 00:00:00 2001
From: Anson Huang <b20788@freescale.com>
Date: Thu, 4 Dec 2014 10:05:38 +0800
Subject: [PATCH 0351/1594] MLK-9955-5 arm: dts: imx6sx: add sema4 support

commit 199bad4508833cd8d29be4d0ca2e8ac1e0678473 from
git://git.freescale.com/imx/linux-2.6-imx.git

add SEMA4 support for i.MX6SX.

Signed-off-by: Anson Huang <b20788@freescale.com>
---
 arch/arm/boot/dts/imx6sx.dtsi |    7 +++++++
 1 files changed, 7 insertions(+), 0 deletions(-)

diff --git a/arch/arm/boot/dts/imx6sx.dtsi b/arch/arm/boot/dts/imx6sx.dtsi
index a0a364b..7f2f91e 100644
--- a/arch/arm/boot/dts/imx6sx.dtsi
+++ b/arch/arm/boot/dts/imx6sx.dtsi
@@ -1326,6 +1326,13 @@
 				status = "disabled";
 			};
 
+			sema4: sema4@02290000 { /* sema4 */
+				compatible = "fsl,imx6sx-sema4";
+				reg = <0x02290000 0x4000>;
+				interrupts = <0 116 0x04>;
+				status = "okay";
+			};
+
 			uart6: serial@022a0000 {
 				compatible = "fsl,imx6sx-uart", "fsl,imx21-uart";
 				reg = <0x022a0000 0x4000>;
-- 
1.7.5.4

